### PR TITLE
Add new dose3 release dose3.4.0-beta1

### PR DESCRIPTION
**Pull request by abate**
new upstream release. This release contains opam cudf conversion routines .

### CODE REVIEW

File: packages/dose/dose.3.4.1-beta1/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+https://gforge.inria.fr/frs/download.php/file/35382/dose3-4.1-beta1.tar.gz
```
Comment (yallop) [inline packages/dose/dose.3.4.1-beta1/url]: This should be:

`archive: "https://gforge.inria.fr/frs/download.php/file/35382/dose3-4.1-beta1.tar.gz"`

Comment (abate) [inline packages/dose/dose.3.4.1-beta1/url]: ups. fixed. opam lint didn't catch this ...

**Comment by samoht** [comment]
```
#=== ERROR while installing dose.3.4.1-beta1 ==================================#
# opam-version 1.2.2
# os           linux
# command      ./configure --with-ocamlgraph --bindir=/home/travis/.opam/system/bin
# path         /home/travis/.opam/system/build/dose.3.4.1-beta1
# compiler     system (4.00.1)
# exit-code    1
# env-file     /home/travis/.opam/system/build/dose.3.4.1-beta1/dose-7657-d682c6.env
# stdout-file  /home/travis/.opam/system/build/dose.3.4.1-beta1/dose-7657-d682c6.out
# stderr-file  /home/travis/.opam/system/build/dose.3.4.1-beta1/dose-7657-d682c6.err
### stdout ###
# [...]
# checking for ocamlc.opt... (cached) ocamlc.opt
# checking for ocamlopt.opt... (cached) ocamlopt.opt
# checking for dynlink.cmxa... yes
# checking for ocaml... (cached) ocaml
# checking for ocamldep... (cached) ocamldep
# checking for ocamlmktop... (cached) ocamlmktop
# checking for ocamlmklib... (cached) ocamlmklib
# checking for ocamldoc... (cached) ocamldoc
# checking for ocamlbuild... (cached) ocamlbuild
# checking for cppo... no
### stderr ###
# configure: error: You must install the cppo pre-processor to compile dose3
```

**Comment by abate** [comment]
ok, before I go any further. Is there a way to test if the package works on my side and then making a commit ? I guess I've to add a private repository ... can you point me in the right direction ? sorry for the noise. Not a real packager :-/

**Comment by samoht** [comment]
You can simply pin your archive in a fresh switch:

```
opam switch -A system test-dose
opam pin add dose <url-of-your-package>
```

**Comment by abate** [comment]
I think it's now ready ...

**Pull request merged**